SPECS/openblas/openblas.spec

```diff
@@ -15,7 +15,7 @@
 
 Name:           openblas
 Version:        0.3.21
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
For extended -> core moves we must also clean-up the specs a bit, because the bar is lower for the extended specs. Please lint the spec and remove references to distro-specific macros (for instance `rhel` or `fedora`) or things that do not matter for Mariner (like unsupported architectures).